kolibri_explore_plugin/assets/src/views/ContentModal.vue

```diff
@@ -99,7 +99,7 @@
 
   // This is overriding the Kolibri media player plugin:
   ::v-deep .content-renderer {
-    border: 0;
+    border: none !important;
```

Comment:
Oh OK! I couldn't find a way to reproduce this. In any case, please use `0 !important!` or the linter will complain.

Reply:
Ah, that explains why it was `border: 0`, then :b Fixed!